README.md

```diff
@@ -38,6 +38,7 @@ This repository contains multiple implementations of `DelegatingHandlers` for di
 Here is a list of the `DelegatingHandlers` provided.
 
 - [NetworkExceptionHandler](#NetworkExceptionHandler) : Throws an exception if the HttpRequest fails and there is no network.
+- [CacheHandler](#CacheHandler) : Implement simple application caching using instructions from custom HTTP headers.
```

Comment:
Are we limiting ourselves for the future by using such a general name? What would be the name of another cachehandler?

Reply:
Perhaps! I used the `Caching` namespace to be able to opt in our out. It could be changed to `ApplicationCaching` since all caching policies are driven by the app in this recipe (and not by the server).
Similarly, we could also rename the classes to include Application in them. I'm open to other suggestions.